backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -100,7 +101,7 @@ public ApiUser createApiUserNoOkta(String email, PersonName name) {
       IdentityAttributes userIdentity = new IdentityAttributes(email, name);
       ApiUser apiUser = _apiUserRepo.save(new ApiUser(email, userIdentity));
       log.info(
-          "User with id={} created by user with id={}",
+          LoggingMessages.AUDIT_UPDATED_USER_MSG,
```

Comment:
this is not the best for readability, because the string contains positional arguments `{}`, and you dont see that when extracted. A better way would be to wrap the log.info statement with a method and invoke the method instead.

Reply:
resolved!